react/components/molecules/card/card.js

```diff
@@ -0,0 +1,152 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, Text, View, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+
+import { Avatar, Icon, Touchable } from "../../atoms";
+
+import { baseStyles } from "../../../util";
+
+export class Card extends PureComponent {
+    static get propTypes() {
+        return {
+            avatarURL: PropTypes.String,
+            icon: PropTypes.String,
+            iconColor: PropTypes.String,
+            iconFill: PropTypes.String,
+            title: PropTypes.String,
+            text: PropTypes.String,
+            subText: PropTypes.String,
+            style: ViewPropTypes.style,
+            pressable: PropTypes.bool,
+            onPress: PropTypes.func,
+            onLongPress: PropTypes.func
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            style: {},
+            pressable: true,
+            onPress: () => {},
+            onLongPress: () => {}
+        };
+    }
+
+    _avatarStyle() {
+        return {
+            ...styles.avatar,
+            ...this.props.style?.avatar
+        };
+    }
+
+    _iconStyle() {
+        return {
+            ...styles.avatar,
+            ...this.props.style?.icon
+        };
+    }
+
+    _textStyle() {
+        return {
+            ...styles.text,
+            ...this.props.style?.text
+        };
+    }
+
+    _renderCardImageAvatar() {
+        return (
+            <Avatar
+                image={{
+                    uri: this.props.avatarURL
+                }}
+                size={60}
+                style={this._avatarStyle()}
+            />
+        );
+    }
+
+    _renderCardImageIcon() {
+        return (
+            <Icon
+                style={this._iconStyle()}
+                icon={this.props.icon}
+                color={this.props.iconColor}
+                fill={this.props.iconFill}
+                height={60}
+                width={60}
+            />
+        );
+    }
+
+    _renderCardImage() {
+        if (this.props.icon) return this._renderCardImageIcon();
+        if (this.props.avatarURL) return this._renderCardImageAvatar();
+    }
+
+    _renderCardText() {
+        return (
+            <View style={styles.textContainer}>
+                {this.props.title ? <Text style={styles.title}>{this.props.title}</Text> : null}

```

Comment:
Just to simplify, perhaps do:
```suggestion
                {this.props.title && <Text style={styles.title}>{this.props.title}</Text>}

```